packages/mui-material/src/Select/Select.test.js

```diff
@@ -292,6 +292,13 @@ describe('<Select />', () => {
     });
   });
 
+  describe('prop: defaultOpen', () => {
+    it('should be open on mount', () => {
+      const { getByRole } = render(<Select defaultOpen value="" />);
+      expect(getByRole('button')).to.have.attribute('aria-expanded', 'true');
```

Comment:
There's one more thing to correct I haven't mentioned (sorry!):
When the Select is open, the rest of the page has `aria-hidden` set. That's why `getByRole` can't find our button here. To work around this, add 
```suggestion
      expect(getByRole('button', { hidden: true })).to.have.attribute('aria-expanded', 'true');
```

The tests should then pass.